src/components/tipRecords/TipRecord.vue

```diff
@@ -282,6 +285,16 @@ export default {
     }
   }
 
+  .tip__url {
+    margin-left: 1rem;
+    margin-right: 1rem;
+
+    a {
```

Comment:
can you add :hover - underline for this link, please to maintain consistency with the other preview